std_msgs/Header header
bool marker_visibile
int32 num_of_visible_markers
geometry_msgs/Pose global_camera_pose
int32[] marker_ids
geometry_msgs/Pose[] global_marker_poses

